corehq/ex-submodules/phonelog/utils.py

```diff
@@ -191,11 +191,14 @@ def __init__(self, domain, xform):
         self.xform = xform
 
     def send_data(self, url):
-        send_device_log_to_sumologic.delay(url, self.log_subreport(), self._get_header('log'))
-        send_device_log_to_sumologic.delay(url, self.user_error_subreport(), self._get_header('user_error'))
-        send_device_log_to_sumologic.delay(url, self.force_close_subreport(), self._get_header('force_close'))
-
-    def _get_header(self, fmt):
+        send_device_log_to_sumologic.delay(url, self.log_subreport().decode('utf-8'),
```

Comment:
Sounds good to me! I was trying to make the least amount of changes to the existing functions but this would make more clear what I'm trying to do here (send decoded strings to celery)

Reply:
Yeah I respect not wanting to disrupt other areas if you don't need to, but in this case I think it is fine to do so. And setting the default to True ensures that the change is still minimal to other usages of those methods that already expect the return value to be encoded.